.github/workflows/actions.yml

```diff
@@ -19,63 +19,31 @@ jobs:
           clang-format --dry-run -Werror *cpp *hpp
         done
   build-on-ubuntu:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/nextsimhub/nextsimdg-dev-env:latest
     steps:
       - uses: actions/checkout@v3
-      - name: Install dependencies
-        shell: bash
-        run: |
-          sudo apt-get install -y cmake libboost-program-options-dev libhdf5-mpi-dev libcurl4-nss-dev
-          git clone https://github.com/catchorg/Catch2.git
-          cd Catch2
-          cmake -DBUILD_TESTING=OFF -Bbuild -S.
-          sudo cmake --build build --config Release --target install
-          cd ..
-          git clone https://github.com/trilinos/Trilinos.git
-          cd Trilinos
-          cmake \
-          -DTPL_ENABLE_MPI:BOOL=ON \
-          -DTrilinos_ENABLE_ALL_PACKAGES:BOOL=OFF \
-          -DTrilinos_ENABLE_Zoltan:BOOL=ON \
-          -DTrilinos_ENABLE_Fortran:BOOL=OFF \
-          -DZoltan_ENABLE_EXAMPLES:BOOL=OFF \
-          -DZoltan_ENABLE_TESTS:BOOL=OFF \
-          -DBUILD_SHARED_LIBS=ON \
-          -Bbuild -S.
-          sudo cmake --build build --config Release --target install
-          cd ..
-          git clone -b checkpoint.1.12.3 https://github.com/Parallel-NetCDF/PnetCDF.git
-          cd PnetCDF
-          autoreconf -i
-          CXX=mpicxx CC=mpicc ./configure --disable-fortran --enable-shared
-          make
-          sudo make install
-          cd ..
-          git clone -b v4.8.1 https://github.com/Unidata/netcdf-c.git
-          cd netcdf-c
-          cmake \
-          -DENABLE_PNETCDF=ON \
-          -DCMAKE_C_COMPILER=mpicc \
-          -DCMAKE_CXX_COMPILER=mpicxx \
-          -DCMAKE_BUILD_TYPE=Release \
-          -DENABLE_TESTS=OFF \
-          -DENABLE_PARALLEL_TESTS=OFF \
-          -Bbuild -S.
-          sudo cmake --build build --config Release --target install
-          cd ..
```

Comment:
It's so nice to remove all of this :sweat_smile:

Reply:
Agreed!